applications/prompt-keda-lsstcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -74,8 +75,8 @@ prompt-keda:
 
   keda:
     minReplicaCount: 3
-    # TODO: this is scaled for SingleFrame, with 25% margin
-    maxReplicaCount: 1100
+    # Scaled for SingleFrame, with 25% margin, would be 1100
```

Comment:
Not sure these kinds of backup comments are needed (we can just `git revert` later), but they also do no harm.